ws-nextjs-app/pages/[service]/send/[id]/FormScreen/index.tsx

```diff
@@ -66,7 +65,7 @@ export default function FormScreen({
             dangerouslySetInnerHTML={{ __html: privacyNotice }}
             css={styles.privacyNotice}
           />
-          {!submitted ? <Submit /> : <Loader />}
+          {!submitted && <Submit />}
```

Comment:
Should we keep the submit button the screen even after the user has clicked it? Not sure if it would affect UX/a11y by removing it, although I guess they should be immediately taken to the 'uploading/error' screen anyway.